packages/embed-chart/jest.config.cjs

```diff
@@ -0,0 +1,13 @@
+const baseConfig = require('../../jest.config.base.cjs');
+const packageJson = require('./package');
+
+module.exports = {
+  ...baseConfig,
+  displayName: packageJson.name,
+  resetMocks: false,
+  transform: {
+    // Using ts-jest (even with its babelConfig option set) causes
+    // problems with some code-studio tests
+    '.(ts|tsx|js|jsx)': 'babel-jest',
+  },
```

Comment:
This can be removed. I noticed when I was switching to Vite that it's not needed since we switched the default from `ts-jest` (tests w/ type check) to `babel-jest` at the root.